frontend/package.json

```diff
@@ -25,7 +25,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
I think that this flag may break the pipeline. Would it be possible to remove it?

Reply:
ok :)